src/main/webapp/components/Footer.js

```diff
@@ -1,8 +1,8 @@
-import React from 'react'
-import ReactDOM from 'react-dom'
-import { a, div } from 'react-hyperscript-helpers'
+import { Component } from 'react'
+import { a, div, hh, footer } from 'react-hyperscript-helpers'
+import { styles } from '../util/ReportConstants';
 
-export class Footer extends React.Component {
+export const Footer = hh(class Footer extends Component {
```

Comment:
nit: This doesn't need to be a full class - it can be a simple function or a functional component.